app/forms/state_file/az_subtractions_form.rb

```diff
@@ -10,8 +10,19 @@ class AzSubtractionsForm < QuestionsForm
 
     def save
       attributes_to_save = attributes_for(:intake)
-      attributes_to_save[:tribal_wages] = nil if tribal_member == "no"
-      attributes_to_save[:armed_forces_wages] = nil if armed_forces_member == "no"
+
+      if tribal_member == "no"
+        attributes_to_save[:tribal_wages] = nil
+      else
+        attributes_to_save[:tribal_wages_amount] = tribal_wages
+      end
```

Comment:
i'm confused about this. what is the second case here? `tribal_member` is anything other than "no" and in that case we want to save the old column value to the new column value?

Reply:
yeah.. wondering if under the if tribal_member == "no" i should also set attributes_to_save[:tribal_wages_amount] = nil